### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by StygianStyx** [MrEliasen/twitch-bot-list#39]
# What is this issue about?

- [ x] Remove an account 

# Confirm you checked the accounts are not already on the lists?

- [x ] Yes I checked

### Twitch Account

Please enter the name of the Twitch.tv account(s) in question:
mine. stygian_styx 

### Reason for adding or removing from list:
its mine and im not a bot, your slandering my name, getting me banned from chats when i have done nothing to you nor anyone else.
![2b9a6176cd292b967cb5759c5207c7c1](https://user-images.githubusercontent.com/76650295/133916632-71c2481d-a5c5-4145-9196-44df5b5ea3a7.png)

Please provide as many details and "evidence" as possible.

**Comment by MrEliasen** [comment]
1. I am not slandering your name, the comes from crawling in many channels at the same time. All information is provided by Twitch Insights, for better or for worse.
2. There is no bot list hosting on this repo any more. Only a whitelist and a grey list remains
3. The problem is never whether you are a bot or a human, it is related to whether or not you are in channels for reasons other than viewing. For example, if you lurk in 100+ channels for whatever reason.

If you have been banned by others, it's because they are banning accounts which lurk in 100+ channels, and they would have done so at their own choice. This will hit people without ill intent, but that is why its the choice of the individual channel what they choose to do.

**Comment by StygianStyx** [comment]
they are doing it because the recent hate bot scare, now your starting to get me banned in channels iv ben following for over 5 years simply because my names in a big, all they have to do is look at my profile to see i stream and am a gamer. You are slandering my name.

**Comment by StygianStyx** [comment]
![4b62546ce5ddaaa112e2a711b3c5a3e4](https://user-images.githubusercontent.com/76650295/133935222-f04975ae-f7d3-482f-87a0-cb03a58a2953.png)

so yah, even your 1st argument is invalide

**Comment by StygianStyx** [comment]
You can either remove my name, greylist me whatever, im tired of whatever your doing affecting me.

**Comment by MrEliasen** [comment]
How about you re-read what I replied to your original comment.
What. Do. You. Want. Me. To. Do. About. It.

Personally, I don't care about Twitch as an ecosystem. The website was made back when I was using it. People asked me to keep it online so all I have done is now make it so the lists are not even loaded from this repository to further separate myself from it. Only it does not is a whitelist and a grey list.

There is no list any more. There is only a white and grey list. Everything else is from Twitch Insights. I do not control who shows up on that list. I have however changed it so its people who lurk in 100+ channels "currently" according to Twitch Insights who will show up.

That is all I can and will do, I am allow people to PR into a whitelist and a grey list so long they can provide "evidence" that its basically a service/opt-in. Whoever shows up on the list of "bots"/accounts from twitch insights is outside of my control.

Don't take what I said as not caring for your situation, but there is just not much else I can do that what I have already done this weekend.

**Comment by StygianStyx** [comment]
this whole thing is illogical, your banning people simply for being in X amount of channels, doesnt matter what they are doing, if they follow, watch, sub, they are all limped together and exterminated. Fuckin Nazi

**Comment by MrEliasen** [comment]
Just so you know, the changes I made are:
- The list now loads directly from Twitch Insights as the "current online" list, instead of the historical list.
- Only accounts lurking in more than 100+ channels will show up from that list
- Whitelisted accounts are now opt-in to unban as well.

Your [comment](https://github.com/MrEliasen/twitch-bot-list/issues/39#issuecomment-922501862) only proves that you didn't care to read nor understand what I just said, and you are now calling me a nazi because I didn't give you the answer you where looking for.

This exchange is over, clearly.

**Comment by MrEliasen** [comment]
@StygianStyx You know what. I am actually happy to try help you, despite how you have been behaving.
If you want me to add you to the grey list, I am still happy to. However, I fear you might just get more negative attention since your name will now show up in the face of anyone who might use the site.

The only, sincere, suggestion I can give you is try to contact the streams you where banned in to attempt to get unbanned. I would honestly not suggest getting put on the grey list.. I do not think it will actually help your case.

But again.. I do not host any lists of accounts/bots any more. If you need to be removed you can probably contact Twitch Insights and ask. But I suggest you approach with a different tone.

I will close this thread again, permanently, if it turns uncivil again.

**Comment by StygianStyx** [comment]
My main issue with whatever this is is it had a streamer i really like almost cry when they realized they banned me, they was shocked to see a unban request with my name on it, confused as to how it happened they later told me it was due to some auto bot thing they did, when i searched for it this is what i found. 
i maybe shoulda cooled off before messaging but dude its literally what's happening. im being thrown into a category and a flip gets switched and i get gassed with the bots or an other user who uses irc chat.


![Inked9d94ac2ce87ff69cfb29835aedf8b354_LI](https://user-images.githubusercontent.com/76650295/133938081-64e7afb0-240a-444a-a00b-c776518321a8.jpg)

**Comment by MrEliasen** [comment]
Honestly, I do understand. There is just not much I can do my end. I have made an additional note on the website trying to explain that there are false positives and they should really avoid the auto list and instead upload their own.

Other than that, I am afraid I cannot really do much. I would consider turning off Twitch Insights and just allow an list to be uploaded but at the same time I don't want to do that as people do have their reasons for banning lurker accounts.

Anyone who gets banned using my website, it should tell them in the ban-reason it was done using it.

Glad you got unbanned however, I think requesting unbans from the streams you watch is the only real *action* available.

**Comment by sandtaster** [comment]
You're in almost 3,000 channels as of me making this post according to Twitch Insights. Unless you can give a plausible and reasonable excuse as to why that is (and believe me, I am _dying_ to hear it), you deserve to be labelled a bot and potentially banned from the thousands of channels of which you provide literally zero value to.

Whether it's harvesting user information, farming channel points/currency, or collecting stats, your behavior is bot-like in nature. You CAN NOT be interacting with/chatting/watching that many channels at once. Nobody can. Your screenshots don't really prove anything either, but I shouldn't need to explain why.

These streamers are using a ban list for a reason, and it's to get rid of accounts like yours from their view lists. Maybe direct your hatred their way instead of at the maker of this tool because I don't believe @MrEliasen is at fault here.

**Comment by LucentW** [comment]
Adding my two cents to the issue.

While it's understandable that viewerlist _abuse_ is widespread and lately it's the "entry point" for the automated hate raids stuff, all the analytics, user info harvesting and stats collecting can be done even by polling the chat without logging on at all (i.e. with tmi.js), rendering the effort of banning those accounts just a purely cosmetic reason.

Going in the clear by logging in it's more like an easy way to "silently" advertise by attracting people that will check out who's in chat, click on the username and check out the profile which might be hosting someone else, actively streaming or having links on their description (such as the whole "streamers' discord" accounts).

BTW: channel points can be farmed only by actively watching the stream, sitting into the chat via IRC/popout won't award you any bonus (except for the _currency_ awarded by the various StreamElements/StreamLabs token mechanisms). The only complaint would be that Twitch's algorithm for random gifted subs _might_ pick one of those accounts, but in the last year I never seen that happening, probably by automatically reducing the chances for massjoiners automatically. One of the screenshots from @StygianStyx shows that they disabled getting subs for channels they don't follow, which backs at least their "no harm intended" over their masslurking.

Having the greylisting-on-request approach, while being some sort of proscription list which will generate some frowning, it's for sure way better than outright banning people that at least shows up here to act about it. The note on the website is enough to explain what the greylist is and people can and should decide whether or not banning the accounts there.

**Comment by MrEliasen** [comment]
I am not getting involved or care for what the latest Twitch drama or issues are. This repo and the web tool was made a couple of years ago where all the issues/reasons where legitimate.  I just wanted to add some context as people seem to forget things might have existed before whatever drama or issues happen which suddenly put them in the spotlight.

With that said, I do believe that the current integration with whitelists, grey list and Twitch Insights is obsolete, and I would be more than happy to kill that off and merely serve the website as a place to upload your own list of accounts to ban instead.

At the end of the day, the tool did serve as specific purpose, but clearly most of those points have become moot so I would be more than happy to get everyones feedback on how to proceed.

I want to keep the tool online, but how it is used could change.

Based on this discussion, I think strip back integrations and leave only the "upload your own list" feature is the way to go.

Thoughts (everyone)?

**Comment by StygianStyx** [comment]
hmm i wonder if there is a way you could maybe add a separate category or something that people can opt into showing they are just users in IRC chat and people can choose to ban them or not with a check mark. I just don't like being lumped together with all the bots. now i do love bots but there are to many people using them for the wrong purpose and its caused a damn Ai scare.

Bots are like kids or animals, people like their own but hate all the rest. lol


maybe add big colorful icon that makes you wanna click it for the upload your own list part so people would be more inclined to use it.

im honestly not sure. i wasn't expecting you do do big changes i just wanted my name out the list.

**Comment by MrEliasen** [comment]
All good! Clearly it was a discussion which was needed haha.

I made the tool to help the many, but if many are now caught in a crossfire and the original rationale for the tool has become obsolete, then its a discussion worth having what to do and how to best adapt it to fit "today"

For everyone, please do post your thoughts/feedback (https://github.com/MrEliasen/twitch-bot-list/issues/39#issuecomment-925717246)

**Comment by sandtaster** [comment]
@StygianStyx is yet to explain why he's in (now) 3,300 channels but _totally isn't a bot_. Meanwhile, his opinion is openly sought after the abusive messages he sent earlier in this conversation. Where will the charity end?

>I just don't like being lumped together with all the bots

So why are you botting then? Moreover, why don't you create a second account to do whatever it is you're doing so that you're "main" account isn't flagged?

Nothing has changed between today and when this tool was created, irrespective of any Twitch drama that's currently happening. In fact, a tool like this would be _helping_ in such a situation, not the other way around.
It still serves the same purpose it always has, and that's helping to remove non-essential bots from streamers' chat. Plain and simple.

And it's no wonder that many streamers choose to use it. Having your chat list full of bots can have detrimental psychological effect on smaller streamers that struggle for even a couple of views (up to 100% of the accounts in the viewer list in some cases).
Being "featured" on some automated account that supposedly helps smaller streamers out with "exposure" is nothing but a scam, especially when streamers have no way to opt-out of activity like that.
The fact that this is an opt-in tool in the first place says it all.

Twitch Insights currently has a list of 5,097 entries on its Known Bots List and there are 29 whitelisted accounts in this list. That's 0.0056%. And no streamer needs all 29 of those bots whitelisted in the first place, maybe a few at most. It just shows how few bots provide an actual service and are useful on Twitch versus almost 100% of the rest of them that provide literally zero value to a streamer/viewer. **Nothing is lost by using this tool.**

**Should anything be changed in my opinion?** Get rid of the greylist. You're either a bot and on the list, or you aren't. And if you're in more than 100 channels at once you should be on the blacklist. End of story.
Keeping the whitelist updated with _actual_ legitimate services should be a big priority, but only if their usefulness can be documented and verified (and that doesn't include you, StygianStyx).

**Comment by LucentW** [comment]
> Having your chat list full of bots can have detrimental psychological effect on smaller streamers that struggle for even a couple of views (up to 100% of the accounts in the viewer list in some cases).

This brings up another relevant issue, Twitch has had a low consideration of small streamers, relegating them to the bottom of the search results. Stuff is changing a bit, as in they're changing the order on search results, but the situation is unfortunately still pretty bad for newcomers, which are resorting to any way possible to get at least to affiliate and kickstart their streaming endeavours.

So, what if @StygianStyx is a small streamer resorting to this to advertise their own stream? While doing so is debatable if correct or not, does this make them worthy of being banned on the spot? Sure it's the streamer choice to decide who can and can't stay on their viewer list -- but ignoring the direct consequences of automatic blocklisting is IMHO way worse than acknowledging via greylisting and putting the disclaimer. If someone doesn't want @StygianStyx to pop up on their viewerlist, they can just add the flag on the greylist and call it a day.

Stay assured that 99.9% of the viewerlist lurkers listed on TwitchInsights won't ever pop an issue here to get greylisted. While @StygianStyx overreacted to @MrEliasen and could've explained _why_ they're doing so, I'd give them the benefit of the doubt on the grounds that they reacted that way because they found themselves banned from a bunch of streams, including one they care about and are actively watching.

Again, it's just my two cents and I'm not here to change your idea (@sandtaster). Regardless of the result of this discussion, if someone doesn't agree with the outcome, they can fork the repository and tailor the software to their own taste.

**Comment by MrEliasen** [comment]
For me personally, I don't have any horse in the race. I am happy to implement whichever outcome which seems to be the right one, from this discussion. I will leave this thread open for a while to get as much info as possible.

Appreciate everyone keeping it civil, I would appear its a fairly heated topic at the moment so thank you.

**Comment by StygianStyx** [comment]
I actually am a smaller streamer. iv only done 3 streams since getting affiliate and Lucent's theory about why is actually spot on. 
Twitch actually changed the way hosts work recently to making it even harder for smaller streamers.
Being in a chat list hurts no one, how do you think i learned how to even use twitches dev site? one of the "bots" in my chat actually responded to me and we became friends. likewise i respond to my chat and make friends. pure and simple its just another way to network.

**Comment by sandtaster** [comment]
>So, what if @StygianStyx is a small streamer resorting to this to advertise their own stream? While doing so is debatable if correct or not, does this make them worthy of being banned on the spot? 

Yes, it does. If you truly believe that any streamer on Twitch (big or small) would be okay with having someone bot in their stream as a means to advertise their own stream (directly or not) you need to get your head checked. _This is NOT okay, this is one of the reasons why the tool exists and is being used._ It doesn't matter _how_ it's being done, it's being done, and @StygianStyx has just openly admitted the fact. Took a while, but we got there in the end.

>If someone doesn't want @StygianStyx to pop up on their viewerlist, they can just add the flag on the greylist and call it a day.

Or unblock him when he cries about it to them directly. Of perhaps the dozen-or-so channels he frequents (and is actually recognized by username) a few at most would even know of this tool's existence, let alone actually use it.
Why not just let every bot creator show-up here as a "human" and complain until they're put on the greylist? You're setting a precedent if you do this, hence why the greylist isn't working.

>Stay assured that 99.9% of the viewerlist lurkers listed on TwitchInsights won't ever pop an issue here to get greylisted.

Right. So when they do, we just unblock them right? Cause suddenly it's not fair that _some_ bots stay on the list while others don't? This issue for example https://github.com/MrEliasen/twitch-bot-list/issues/21#issuecomment-907644402 "they totally aren't a bot, they're my friend" then both accounts are banned. Not suspicious at all.

**_There can't possibly be any legitimate user accounts caught by Twitch Insights because any human can't be in over 100 channels at once._** I don't know how much clearer this could be. Not all bots are equal, but those that are more important can easily be put on a whitelist like StreamElements (for example).

>Being in a chat list hurts no one

So I wonder why people are using a tool to remove these "people"/bots who constantly lurk in their chat... Hmm.

>[paragraph that added zero to the discussion]

And then you go to the extent to use this very issue to plug your own stream one more time. The fact that you so happily associate yourself with one of the biggest bot accounts (that actually started this whole blocking movement/problem in the first place, ElectricalLongBoard) says it all.

My two cents also @LucentW though I'm getting tired of repeating the same points.

**Comment by StygianStyx** [comment]
Dude is wanting a witch-hunt and to escalate/inflame the thread.

I'm a streamer, I don't mind having IRC users in my chat. its not like they are popping up adds or something. its literally just another name in the chat list. so if someone's actually looking at my Chatlist that hard to notice and click em more power to em, I can actually confirm a lot of streamers don't care if your in the chat list and not bothering anyone if your a real users at the account, as someone who uses IRC and meets many many streamers and I can guarantee you the big streamers care even less about the "issue".  I would know I mod for a couple partners and watch/host some bigger streamers. 

how would you like your name put in a list of people to just auto ban? so you can be banned from any future channels you might find...or accidently by friends/other streamers you watch and when the mods ask them while they are live while YOU are banned they can "pause their stream" to apologize to you several times on the verge of tears and confused af, embarrassing both you and them. 

Its fear mongering. people are using this because the recent hate bot scare, well I can tell you that hate bots wont be in this list. Twitch bans accounts that are doing things against ToS. Nothing about my account is against ToS in fact Twitch support has seen the exact code i use to connect to IRC chat and guess what they did? Unbanned my account.

as stated the guy you call a "bot" is a real dude, we have had many chats, he has helped me and many other streamers on many occasions and his whole channel exists to help smaller streamers. i met him because he was "a bot" in my chat and i went to his channel, as a result i have MANY new friends, am part of a retro game community, a Minecraft community, a just chatting community, a music community, and many more. i have helped said communities with discord bot setup, twitch bot setup, twitch panels, VR avatar stuff, etc, the list goes on and on. 

Its a networking tool pure and simple.

**Comment by sandtaster** [comment]
I'm a streamer too, and I _do_ mind having pointless IRC users in my chat. Clearly you aren't the target audience for this tool, so I'm not sure why you feel your opinion should be held to such high regard. You very clearly gravitate toward other like-minded individuals (bot makers and members of such "communities") so of course they're going to side with you on the issue if asked. They may even feel personally attacked, like you.
Meanwhile, the average Twitch streamer (I say "average", but what I mean is "vast majority") who has no prior involvement with bot making and/or makers would gravitate to the opinion of them being pointless/useless in their stream, or at the very least would see no harm in removing them. Which is true because there is no harm in doing so.
Big (define "big") streamers don't care about banning bots because they don't have the resources or time to care. If they're that big anyway they aren't going to miss one person from chat in a sea of tens of thousands. You don't mean as much to that streamer you idolize as much as they mean to you, just because they said your username when you donated $5.

>hey sorry about that i think you got caught up in the auto ban thing when i was banning some bots ><

Such a tear-jerking reaction.
Have you asked them why they were using the tool in the first place? Because apparently even some of your friends are also of the opinion that bots and primarily automated accounts have no place in their chat. And so now you're un-banned from the few channels that actually care about you enough to do so... Where's the problem again?
I have no fear of being put on any list because unlike you _I'm not a bot or using my account to bot with in over 3,000 streams at once._ All of this drama you've decided to stir-up could have been avoided if you had just made a second account to run your little scripts with.

I don't think you understand what the term fearmongering even means. You're the one who keeps bringing it up, not me. Of course "hate bots" aren't going to be on the list because they're highly-organized disposable operations that strike semi-randomly. This discussion was never about stopping hate bots, it was about removing bots from chat. All kinds of bots. It's not like this is a foreign subject, nor is it against any kind of rules.
But hey, if you say so. Not hard to submit bleached code when asked. Why was your account banned in the first place? Suspicious activity? You admitted yourself that you're using your bot to advertise your own stream, I'm pretty sure that's not on the list of allowed uses... Maybe this thread could be brought to their attention.

You're getting mixed-up here. Of course he's a real dude, a _human_ had to create the bot. A human has had to create _every_ bot on Twitch at some point. I'm not saying that the person behind that account isn't a real person, nor am I saying that _you're_ not a real person. What I'm saying is that a stream that runs a script 24/7 on an automated loop that provides "exposure" to people who might actually prefer not to be "exposed" _is_ a bot. Just because the human behind the operation decides to rear their head every-so-often doesn't negate that fact, they're still a bot account. 

Here's the difference between you and CommanderRoot: CommanderRoot is a bot account that provides _actual tangible value_ to streamers on Twitch that wish to use his services. They're opt-in tools like this one, and you're able to (and quite within your rights as a streamer) to ban him from your stream if you want.
And would you believe it, there's an actual _real dude_ behind the bot. Mind-blowing stuff.

I'm glad you brought-up ad-blockers earlier in your reply because it's the perfect analogy to all of this. If you're blocking ads you're blocking _all_ ads. You don't say "Oh, I'm never going to buy a Volvo, but I'm just going to let these Volvo ads show up on random sites because I don't mind. Just the Volvo ads. Even though they provide me with no value whatsoever, they're different than all the other ads on the internet somehow, even though they serve the exact same purpose".
People using a catch-all solution are doing so because casting a wider net at times is often the more effective solution than not. If you were on the blacklist by literal mistake this conversation would have ended after your first message, but here we are.

Re-read this https://github.com/MrEliasen/twitch-bot-list/issues/39#issuecomment-922468173 if you need a quick refresher course. This tool is, for all intents and purposes, _**working as intended in its current state**._ Build your own "un-ban StygianStyx" tool, nobody's stopping you. Would be interesting to see how many people use it though...

**Comment by StygianStyx** [comment]
Do you realize what me being a Twitch Coil user means? Me being in a streamers chat list actually means they have a chance to get random free bits. i brought up adblockers because i don't use them, all streamers i visit get paid.

What are you even doing on GitHub with such a fear of Code/Progress? Everyone has their own thing they are good at dude

I already said why and how my friend banned me. They was scared of hate bots and used a tool that had my name in the list not understanding that the tool just lists IRC users. yah the DM sent after stream was pretty chill, however the initial shock of it wasn't, shit like that makes you wonder what mod you cant trust then when you realize it was you yourself it really gets to you..

You being a user who cares if IRC users are in your chat doesn't mean all do, i didn't ask the list be deleted, i asked my name be. i even gave the option of making a separate category with IRC users in it who ask to be put there still leaving you the option of banning me, however i would not be in a list with true bot accounts.

you have never been to ELBs channel have you? its not a script, its a chatroom and when hes not on the road in his van he streams his discord video chat and people can hop in.

https://twitter.com/ELB_Retweets/status/1404508981884821513?s=20

**Comment by TGxNightshade** [comment]
Firstly, I'd like to say I've known @StygianStyx for a while now and know they are most definitely not a bot. Verified through personal conversations via discord, other streamers, and my own stream. Why it could ever be made a case that this individual is a bot is unknown to me. 

Secondly, I feel it must be stated the level of unprofessional indecency that has been shown by @MrEliasen. I guess we should start with the first thing that specifically caught my eye, the "I don't have a horse in this race" comment. If this is the case, you shouldn't even be talking right now. Your decisions mean nothing to you as well as the consequences of those actions, whoever they may be affecting. This is an inadequate way to deflect or pass blame, making it seem like it's not your fault, when your attitude entering the situation was already one of quiet dismissal of anyone else's problems, therefore leading to this problem in the first place and your inability (or more likely unwillingness) to proceed in this manner with any level of appreciation or respect for your fellow humans. I would suggest rethinking this tactic. I'm not sure how much it matters to you (though I'd be willing to bet based on what I've read that it doesn't), but this isn't the way to make friends. I understand this is a very simplistic way of stating this but I hope that you CAN understand the gravity of this statement and how it likely affects your day to day life and the people you interact with (or more likely don't).

Thirdly, I must address @sandtaster. I can only assume this is something either in their early to mid thirties or a teenager, either living under the close care and attendance of a legal guardian or general provider of their necessities. I'd say "someone" but this is obviously a troll, and trolls aren't people. They don't do anything productive for a society nor posses any higher levels of thinking commonly associated with homo sapiens. Living off of forums such as these and other comments sections of social media. They have no friends or true allies. They have no real support. You are a leech and scum. But you already know that.

**Comment by MrEliasen** [comment]
> the "I don't have a horse in this race" comment. If this is the case, you shouldn't even be talking right now. Your decisions mean nothing to you as well as the consequences of those actions, whoever they may be affecting.

Did you even read the entire thread? What is it I have been trying to do with this thread and been encouraging?
Also, how about you stick to the reason I kept this thread open in the first place, and what I asked wouldn't happen?
Let me help you

- I am asking for feedback and suggestions as to how to proceed and have offered my suggestions.
- I have asked for this thread not be become personal, and I see you and a few others in this thread.

It is clear that you are trying to defend one of the people in this thread, commendable, but what you are doing is having the opposite effect. I am about to lock this thread if it does not steer clear of the personal/directed comments.

This goes for **everything**, thank you.

> This is an inadequate way to deflect or pass blame, making it seem like it's not your fault, when your attitude entering the situation was already one of quiet dismissal of anyone else's problems, therefore leading to this problem in the first place

You are right, I did go into this with a dismissal in mind, because i have been getting quite a few messages about this over the years and in a few number of these instances it has been quite "colourful" language directed at me.

In this case, which you would have seen if you read the entire thread, I did re-estimate the communications and even after being called a literal nazi I still though "it wasn't fair and this person is just frustrated, I should investigate further" and reopened the thread and has been quite sympathetic going forward and your inability (or more likely unwillingness) to proceed in this manner with any level of appreciation or respect for your fellow humans

You are projecting your own assumptions and opinions as fact.

> I would suggest rethinking this tactic. I'm not sure how much it matters to you (though I'd be willing to bet based on what I've read that it doesn't), but this isn't the way to make friends. I understand this is a very simplistic way of stating this but I hope that you CAN understand the gravity of this statement and how it likely affects your day to day life and the people you interact with (or more likely don't).

This is a simple and obsolete repository which have become a bit heated suddenly due to current events on Twitch.
What you are trying to do with this statement is discredit me and my opinions with presenting me 2 options of response; Either I "submit" to what you said, in which case you prove me wrong, or I reject what you said, in which case you have already projected how people should perceive this as also showing you as being correct and me being wrong.

Pathetic.

---

@sandtaster @StygianStyx @TGxNightshade I will lock this thread if you continue to make it personal. Please re-read this comment: https://github.com/MrEliasen/twitch-bot-list/issues/39#issuecomment-926785573

Post your suggestions on what a possible solution could be, instead of whatever the hell this thread has degenerated to.

If you have a suggestion, you can discuss it here, or make a new issue where you describe your suggested solution so other might comment directly on that idead.

The drama stops now.

**Comment by LucentW** [comment]
Quoting two parts of different comments from @sandtaster about the reasoning there might be behind the entire influx and on why using the tool:

> So I wonder why people are using a tool to remove these "people"/bots who constantly lurk in their chat... Hmm.

> I don't think you understand what the term fearmongering even means. You're the one who keeps bringing it up, not me. Of course "hate bots" aren't going to be on the list because they're highly-organized disposable operations that strike semi-randomly. This discussion was never about stopping hate bots, it was about removing bots from chat. All kinds of bots. It's not like this is a foreign subject, nor is it against any kind of rules.

The growing traffic of people resorting to automated bots ban tools **is** because of those disposable accounts' operations. We all know it's not a solution for that issue because of the fast flux the operators use/used in their trolling (I don't even know if the stuff is still a thing), _but_ it has been tackled by other developers with specific automated moderation bots to curb throwaway accounts flooding slurs (i.e. sery_bot), therefore even more so it should be clear on the disclaimer that the tool's target is people that want to cleanup their lists from bots and users massjoining stream chats.

The entire hate raids drama anyways **is** fearmongering, especially when people start spreading narratives and non-solutions. The fear generated and the consequences look pretty much like how _some_ people are tackling the current pandemic... (_cough_ 5G and ivermectin users _cough_)
_Necessary disclaimer: I'm not negating the usefulness of the tool -- but it's not what it tackles. To put it on the same scope of the previous assertion, ivermectin is useful to treat parasites, but not viruses._

I still stand for the greylisting/disclaimer approach, it's not elegant but it's respectful enough for both sides. If someone wants to hammer 'em all, including greylisted accounts, they're a flag away from doing so (as long as a select all option is going to be made available on the greylist too).

Needless to say, the already quoted CommanderRoot should be moved to the greylist too.

**Comment by MrEliasen** [comment]
@LucentW Do not turn this discussion political or use political examples, this thread is already enough of a tinderbox as is.
Also if you believe commanderroot should be grey listed, just create a PR, create an accompanying issue and fill out the fields 👍🏻.

I have never stated that I am the sole arbiter of what does and does not go on the list. Only the whitelist is a bit more specific as it should only be for open opt-in bots which everyone can use.

Just to confirm, since it looks to me like many are basing suggestions purely from this repository's perspective. Should the tool continue to use the Twitch Insights (with a grey/whitelist) or just allow a custom list to be uploaded?

Judging by how polarising this tool has become, I just want everyone to understand that changes can be made outside of just how or why these lists should work/exist.

As I have said before, I don't really care what it ends up looking like, so long the tool is at least available to upload a list manually.

Thanks.

**Comment by MrEliasen** [comment]
I have closed this thread, its unmanageable.
Please continue the discussion here: https://github.com/MrEliasen/twitch-bot-list/issues/40
